linkerd/protocol/http/src/e2e/scala/io/buoyant/linkerd/protocol/HttpEndToEndTest.scala

```diff
@@ -352,6 +350,36 @@ class HttpEndToEndTest extends FunSuite with Awaits {
     assert(!headers.contains("dtab-local"))
   }
 
+  test("dtab-local header is ignored") {
+    val stats = NullStatsReceiver
+    val tracer = new BufferingTracer
+
+    @volatile var headers: HeaderMap = null
+
+    val dog = Downstream.mk("dog") { req =>
+      headers = req.headerMap
+      req.response
+    }
+    val dtab = Dtab.read(s"""
+      /svc/* => /$$/inet/127.1/${dog.port} ;
+    """)
+
+    val linker = Linker.Initializers(Seq(HttpInitializer)).load(basicConfig(dtab))
+      .configured(param.Stats(stats))
+      .configured(param.Tracer(tracer))
+    val router = linker.routers.head.initialize()
+    val server = router.servers.head.serve()
+
+    val client = upstream(server)
+
+    val req = Request()
+    req.host = "dog"
+    req.headerMap.set("dtab-local", "/a=>/b")
+    await(client(req))
+    assert(headers("dtab-local") == "/a=>/b")
+    assert(!headers.contains(dtabWriteHeader))
+  }
+
```

Comment:
does this test verify that dtab-local does NOT impact the l5d-dtab?  (i.e. we should check the l5d-dtab-ctx or something?)